.ci/images_matrix.json

```diff
@@ -1,5 +1,32 @@
 {
-    "android_api": [ 28, 29, 30],
-    "android_ndk": ["--android-ndk", ""],
-    "gcloud": ["--gcloud", ""]
+    "android_api": [
+        {
+            "api": 28,
+            "build_tools": "28.0.3"
+        },
+        {
+            "api": 29,
+            "build_tools": "29.0.3"
+        },
+        {
+            "api": 30,
+            "build_tools": "30.0.3"
+        },
+        {
+            "api": 31,
+            "build_tools": "31.0.0"
+        },
+        {
+            "api": 32,
+            "build_tools": "32.0.0"
+        }
```

Comment:
Until now, we could build old APIs using the latest build tools version, is it not the case anymore ?

Reply:
It is because of the JDK 8 feature. Latest build tools requires latest version of Gradle plugin. But since JDK 8 only works on Gradle versions prior to V7, it would break compatibility.
We could use latest build tools which does not break compatibility ([28 => 30.0.3, 29 => 30.0.3, 30 => 30.0.3, 31 => 31.0.0 or 32.0.0, 32 => 32.0.0]). What do you think ?

Reply:
I think it would be easier this way yes 👍